css-forms-1/Overview.bs

```diff
@@ -837,6 +854,16 @@ ISSUE: Refine meter, progress, switch and range input styling.
   width: 1em;
   height: 100%;
 }
+
+input[type=checkbox][switch]::slider-thumb {
+  border-radius: 100%;
+  height: 1em;
+  position: absolute;
+}
+
+input[type=checkbox][switch]:checked::slider-thumb {
+  inset-inline-end: 0;
+}
```

Comment:
I wonder if we could use grid and align properties somehow here.

Reply:
I suspect we'll still end up using absolute positioning for range inputs though. So it depends how much cleaner it actually is.